network-api/networkapi/wagtailpages/factory/__init__.py

```diff
@@ -28,7 +31,9 @@ def generate(seed):
     participate_page_featured_highlights.generate(seed)
     styleguide.generate(seed)
     youtube_regrets_page.generate(seed)
-
+    areas_of_focus.generate(seed)
+    homepage_areas_of_focus.generate(seed)
+    homepage_take_action.generate(seed)
```

Comment:
not a huge deal but let's order these alphabetically since it looks like that's what we have been trying to do here